src/components/Table/Table.tsx

```diff
@@ -152,6 +152,7 @@ export class Table<I extends TableDataItem = Record<string, string>> extends Rea
             content = id;
         }
 
+        if (id === '_selection') return content;
```

Comment:
did this to remove double wrapping of checkbox so `padding: inherit` will work

Reply:
But now selection hoc "data" leaked to the base Table. Let's find a workaround here

Reply:
this is one with additional wrapper
<img width="682" alt="Screenshot 2023-10-10 at 21 25 18" src="https://github.com/gravity-ui/uikit/assets/79052139/ae6b40ee-0af7-4859-bc8e-49abc58c7f4b">

this is one without
<img width="701" alt="Screenshot 2023-10-10 at 21 26 24" src="https://github.com/gravity-ui/uikit/assets/79052139/793473f0-9843-4721-9b77-ab281cbb6d9d">

Reply:
If I add `padding: inherit` to the `th-content` class, I can fix the existence of the extra wrapper, but then all the other cells are broken
<img width="730" alt="Screenshot 2023-10-10 at 21 27 38" src="https://github.com/gravity-ui/uikit/assets/79052139/ac93df1c-8d9f-4de8-8130-a4a66e19b79e">

Reply:
i can add different wrapper for cell with `id === '_selection'`, something like this:
`if (id === '_selection') return <span  className={b('th-content_selection')}>content</span>;`

Reply:
I think we can remove this wrapper (`th-content`) completely, and add these styles instead:
```scss
    &__head &__cell {
        @include mixins.text-accent;
        &::first-letter {
            text-transform: uppercase;
        }
    }
```

Reply:
> I think we can remove this wrapper (`th-content`) completely, and add these styles instead:
> 
> ```css-scss
>     &__head &__cell {
>         @include mixins.text-accent;
>         &::first-letter {
>             text-transform: uppercase;
>         }
>     }
> ```

like [this](https://github.com/gravity-ui/uikit/pull/654/commits/af2ecc18111eb3e5907729e27dd091117f70a923) ?

Reply:
@amje